.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.3' # Replace this with the minimum Julia version that your package supports. E.g. if your package requires Julia 1.5 or higher, change this to '1.5'.
+          - 'min' # Minimum supported version according to the Project.toml file.
           - '1' # Leave this line unchanged. '1' will automatically expand to the latest stable 1.x release of Julia.
           - 'lts'
```

Comment:
At this point we can remove this line, which is redundant with the "min" version
```suggestion
```